packages/trader/src/App/Components/Elements/PositionsDrawer/positions-drawer.jsx

```diff
@@ -11,7 +11,6 @@ import EmptyPortfolioMessage from 'Modules/Reports/Components/empty-portfolio-me
 import { connect } from 'Stores/connect';
 import { getContractTypesConfig } from 'Stores/Modules/Trading/Constants/contract';
 import { isCallPut } from 'Stores/Modules/Contract/Helpers/contract-type';
-import { filterByContractType } from './helpers';
```

Comment:
Can we reuse this instead of adding it internally?

Reply:
done